webrtc-android-armeabi-v7a.pc

```diff
@@ -0,0 +1,18 @@
+pc=${pcfiledir}
+
+Name: webrtc
+Description: webrtc
+Version: 1
+
+Cflags: \
+	-I${pc}/include \
+	-D_GLIBCXX_USE_CXX11_ABI=1 \
```

Comment:
`-D_GLIBCXX_USE_CXX11_ABI=1 ` can / should be removed now.

Reply:
:+1: